### ISSUE TITLE
Multiselect odd behavior with virtualized dataSource and k-no-value-binding.bind="false" or not present

### ISSUE THREAD

**Issue by pKrav75** [aurelia-ui-toolkits/aurelia-kendoui-bridge#687]
Navigate to the virtualized gist for multiselect https://gist.run/?id=c554a5d7cc6264fdb9022fcd274ed637
Remove initial value and k-no-value-binding settings
```javascript
                    k-no-value-binding.bind="true"
                     k-value.bind="[10265, 10289]"
```

Select a first value and then scroll down in the list using the arrow down key.
It seems that scrolling is bound to the initial page and  adds continuously the same value in the selected items as shown here 
![image](https://cloud.githubusercontent.com/assets/17618403/22028930/2e791d06-dcd9-11e6-924b-71bbb3a01092.png)

Restore the k-no-value-binding.bind="true" and the controls works ok.

So this issue here is to understand what is the exact meaning of this setting (not documented as far as i know) and ideally if we can get rid off it.

Thx

**Comment by JeroenVinke** [comment]
@xenod do you have time to look into this bug?

**Comment by JeroenVinke** [comment]
Should be resolved in 1.2.0
